kratos/utilities/global_pointer_utilities.h

```diff
@@ -180,7 +180,7 @@ class GlobalPointerUtilities
         if(current_rank == master_rank)
         {
             std::sort(all_remote_ids.begin(), all_remote_ids.end());
-            std::unique(all_remote_ids.begin(), all_remote_ids.end());
+            static_cast<void>(std::unique(all_remote_ids.begin(), all_remote_ids.end()));
```

Comment:
yikes :sweat_smile: 
I think you should rather fix it properly, I am pretty sure whoever implemented `[[nodiscard]]` here had their reason

@RiccardoRossi can you have a look please?

Reply:
Yep, just to remark that my proposal is actually the one above:

```
auto last = std::unique(all_remote_ids.begin(), all_remote_ids.end());
all_remote_ids.erase(last, all_remote_ids.end());
```